packages/ui/src/components/EventsCard/EventsCard.mocks.ts

```diff
@@ -0,0 +1,33 @@
+export const MAIN_TITLE = 'Najbliższe terminy';
+
+export const BUTTON_TEXT = 'Przejdź do wydarzenia';
+
+const eventDescription =
+  'Sed dui elementum, elit est id malesuada leo varius. Arcu tempus, non justo leo egestas.' +
+  'A cras fames a ullamcorper. Laoreet ut orci semper massa integer ut consequat aliquam. Commodo enim, sed' +
+  'lorem nibh vivamus neque congue...';
+
+export const EVENTS = [
```

Comment:
💭 As a convention we should use UPPER_CASE only for variables with primitive values

Reply:
it's done